### DISCUSSION TITLE
QS-Zigbee-C01 works when paired but suspends reachability one or more days later

### DISCUSSION THREAD

**Discussion by GerhardLang** [Koenkk/zigbee2mqtt#9535] [Devices]
I do have several Lonhonso QS-Zigbee-C01 which can be paired and calibrated without problem.

My environment: Synology with Conbee II, zigbee2mqtt, homebridge.

In the Zigbee2mqtt configuration of the device details page all works fine at first, in homebridge/homekit, too.
When the device decides not to respond anymore, all is not working anymore.

Actually I do have error messages from two devices (others do not pop errors but only do not work anymore) (see errors).
when I do a new pairing it works again for a time. 

Any idea, is it a qos topic, are they instable, what else could it be? thank you for hints.

ERRORS:
Publish 'get' 'position' to 'Studio Rolladen vorne' failed: 'Error: Read 0xa4c1383a197aa77d/1 closuresWindowCovering(["currentPositionLiftPercentage"], [{"sendWhenActive":false,"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false})]([url](url)) failed (no response received)'
Error Publish 'get' 'position' to 'Studio Rolladen mitte' failed: 'Error: Read 0xa4c1388067c61b7f/1 closuresWindowCovering(["currentPositionLiftPercentage"], {"sendWhenActive":false,"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'

![image](https://user-images.githubusercontent.com/89694701/140620011-81e2d26e-e156-4342-a01b-39491be63102.png)

**Comment by Koenkk** [comment]
Looks the same as #9249

**Reply by GerhardLang** [reply]
Thank you, very similar. I will add my comment there an waiting for solution.
